### PR TITLE
Sync wikibase string lengths with store

### PR DESCRIPTION
**Pull request by toban**

### CODE REVIEW

File: src/store/wikis.js [hunk 1]
```diff
@@ -58,10 +58,22 @@ const mutations = {
     const federatedPropertiesSetting = details.public_settings.find(setting => setting.name === 'wikibaseFedPropsEnable')
     const wikibaseFedPropsEnable = federatedPropertiesSetting ? parseInt(federatedPropertiesSetting.value) === 1 : false
 
+    const wwWikibaseStringLengthStringSetting = details.public_settings.find(setting => setting.name === 'wwWikibaseStringLengthString')
```
Comment (toban) [inline src/store/wikis.js:61]: this looks like puke

File: src/components/Pages/ManageWiki/Cards/Wikibase.vue [hunk 2]
```diff
@@ -68,18 +85,17 @@ export default {
...
 
       Promise.all(promises)
         .then(() => {
           alert('Update success!')
-          this.$router.go()
         })
         .catch(err => {
           console.log(err.response)
           alert('Something went wrong.')
-          this.$router.go()
```
Comment (addshore) [inline src/components/Pages/ManageWiki/Cards/Wikibase.vue:82]: Glad to be getting rid of these fun terrible reloads :D

File: src/store/wikis.js (hunk 1 above)
Comment (addshore) [reply src/store/wikis.js:61]: I think at some point we would want to make the API return the default values.
Then most of this puke can go and just use whatever the API would send.
This puts the control of the defaults etc in the API as well which is probably where it belongs.

**Review by addshore — APPROVED** [review]
Seems to be working!
(but could also do with a tooltip about update lag)

**Review by jakobw — APPROVED** [review]
LGTM :sparkles:

**Pull request merged**